scenario/src/main/kotlin/com/cosmotech/scenario/azure/ScenarioServiceImpl.kt

```diff
@@ -778,6 +782,48 @@ internal class ScenarioServiceImpl(
             CosmosItemRequestOptions())
   }
 
+  private fun sendScenarioMetaData(
+      organizationId: String,
+      workspace: Workspace,
+      scenario: Scenario
+  ) {
+    if (workspace.sendScenarioMetadataToEventHub != true) {
+      return
+    }
+
+    val eventBus = csmPlatformProperties.azure?.eventBus!!
+    val eventHubNamespace = "${organizationId}-${workspace.key}".lowercase()
```

Comment:
Test here workspace.useDedicatedEventHubNamespace and if false log a WARNING: "workspace must be configured with useDedicatedEventHubNamespace to true in order to send metadata"